docs/cmdline-opts/location.md

```diff
@@ -22,9 +22,10 @@ location (indicated with a Location: header and a 3XX response code), this
 option makes curl redo the request on the new place. If used together with
 --include or --head, headers from all requested pages are shown.
 
-When authentication is used, curl only sends its credentials to the initial
-host. If a redirect takes curl to a different host, it does not get the
-user+password pass on. See also --location-trusted on how to change this.
+When authentication or cookie is used, curl only sends its credentials to the
+initial host. If a redirect takes curl to a different host, it does not get the
+user+password or cookie pass on. See also --location-trusted on how to change
+this.
```

Comment:
This needs to clarify the cookie use. Because normally cookies are only sent to the correct host names so this extra precaution is only for when you "force" cookies explicitly.

Reply:
What do you think about this description?